astro/src/content/docs/lifecycle/authenticate-users/oauth/endpoints.mdx

```diff
@@ -770,6 +770,11 @@ Ensure you are sending these parameters as form encoded data in the request body
     This parameter is optional when the `Authorization` header is provided.
     Please see the [the Client Secret table](#client-secret) for more details.
   </APIField>
+  <APIField name="expires_in" type="Integer" optional since="1.99.9">
+    The expiration time, in seconds, for the access and Id tokens (if the `openid` scope was included) that will be returned in the response.
+
+    When provided on the request, this value will override the default <InlineField>tenant.jwtConfiguration.timeToLiveInSeconds</InlineField> or <InlineField>application.jwtConfiguration.timeToLiveInSeconds</InlineField> values, but it must be less than or equal to those values.
```

Comment:
Same comment as above.

```suggestion
    When provided on the request, this value will override the default TTL, provided it is less than the default value. The default TTL is taken from <InlineField>application.jwtConfiguration.timeToLiveInSeconds</InlineField> if it is set there, otherwise it is taken from <InlineField>tenant.jwtConfiguration.timeToLiveInSeconds</InlineField>.
```